### PR TITLE
feature #1996 - navigation from status author name to profile

### PR DESCRIPTION
**Pull request by goranjovic**
addresses #1996 

### Summary:

Tapping on author name or icon in Discover statuses leads to status author profile.

### Steps:

see #1996 

status: ready

### CODE REVIEW

File: src/status_im/ui/screens/discover/events.cljs [hunk 1]
```diff
@@ -146,3 +146,9 @@
   (u/side-effect!
     (fn [_ _]
       (discoveries/delete :created-at :asc 1000 200))))
+
+(handlers/register-handler-fx
+  :show-status-author-profile
+  (fn [{db :db} [_ identity]]
+    {:db (assoc db :contacts/identity identity)
```
Comment (rasom) [inline src/status_im/ui/screens/discover/events.cljs]: alignment in map

Comment (goranjovic) [reply src/status_im/ui/screens/discover/events.cljs]: done

**Review by oskarth — APPROVED** [review]

**Review by yenda — APPROVED** [review]

**Comment by goranjovic** [comment]
@rasom Are you happy with the changes? Can we move it to testing?

**Comment by oskarth** [comment]
@goranjovic Seems like a minor stylistic point that you addressed; moved to ready for testing

**Comment by oskarth** [comment]
@annadanchenko @asemiankevich What's the state of this PR? Would you prefer it if we just merged it or do you think you'll get to it soonish?

**Comment by asemiankevich** [comment]
i have tested this and it works the following way:
1. I can navigate to profile when clicking user name
2. I can navigate to profile when clicking user picture

The issue with empty profile is reflected in https://github.com/status-im/status-react/issues/2306. I think we can merge this one

**Pull request merged**